src/stream/src/executor/aggregation/mod.rs

```diff
@@ -356,13 +360,23 @@ pub async fn generate_managed_agg_state<S: StateStore>(
             keyspace.clone()
         };
 
+        let state_table = Arc::new(Mutex::new(StateTable::new(
```

Comment:
A less important question: should we reuse the state table among multiple agg calls, or one state table per agg call?